src/main/java/org/iban4j/bban/BbanStructure.java

```diff
@@ -93,6 +93,12 @@ private BbanStructure(final BbanStructureEntry... entries) {
                         BbanStructureEntry.branchCode(4, 'n'),
                         BbanStructureEntry.accountType(2, 'n'),
                         BbanStructureEntry.accountNumber(8, 'c')));
+        
+        structures.put(CountryCode.BY,
+                new BbanStructure(
+                        BbanStructureEntry.bankCode(4, 'c'),
+                        BbanStructureEntry.accountType(4, 'n'),
```

Comment:
I'm looking for documentation that these four digits are in fact the `accountType`.
- https://bank.codes/iban/structure/belarus/ calls it branch code.
- https://www.swift.com/resource/iban-registry-pdf recognizes that they're separate from the account number, but does not indicate what they are.
- [Wikipedia](https://en.wikipedia.org/wiki/International_Bank_Account_Number#IBAN_formats_by_country) calls it "balance account number", but they have the overall format wrong. They have https://www.nbrb.by/payment/IBANBIC as their source; maybe you can help me explore that source?

Reply:
Hello! You are almost right and those sources you have found actual, but there are some differences. 
First of all read please this link that contains actual BBAN table for every country [](https://www.mobilefish.com/services/bban_iban/bban_iban.php)
By the way, I also like you thought that it was "branch code" and manily checked it in this project. I took generated IBANs and tried to check it on this official site [](https://www.iban.ru/iban-checker), but it wasn't valid (the structure incorrect).  
For account type it generates valid IBANs

> I'm looking for documentation that these four digits are in fact the `accountType`.
> 
> * https://bank.codes/iban/structure/belarus/ calls it branch code.
> * https://www.swift.com/resource/iban-registry-pdf recognizes that they're separate from the account number, but does not indicate what they are.
> * [Wikipedia](https://en.wikipedia.org/wiki/International_Bank_Account_Number#IBAN_formats_by_country) calls it "balance account number", but they have the overall format wrong. They have https://www.nbrb.by/payment/IBANBIC as their source; maybe you can help me explore that source?